gallery/visualtests/nxDrawer.js

```diff
@@ -113,6 +113,34 @@ describe('NxDrawer', function() {
     });
   });
 
+  describe('NxDrawer with NxToast', function() {
+    const browserSetup = setupBrowser('#/NxDrawerWithNxToastExample', false);
+    const { getPage, checkFullPageScreenshot, waitAndGetElements, wait } = browserSetup;
+    const openDrawer = openDrawerFromPage(browserSetup);
+
+    const buttonId = 'nx-drawer-with-nx-toast-open-button';
+    const drawerId = 'nx-drawer-with-nx-toast';
+
+    const launchToastsFromPage = async () => {
+
+      const [openToastBtn] = await waitAndGetElements('.nx-btn-bar .nx-btn--secondary');
```

Comment:
you should be able to remove .nx-btn-bar since the openToastBtn is not contained within a bar like the other NxToast examples